### DISCUSSION TITLE
Should we release 0.69.5?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#30] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/31

---

Current Release: [0.69.4](https://github.com/facebook/react-native/releases/tag/v0.69.4)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by wfern** [comment]
Can we pick this one?

https://github.com/facebook/react-native/commit/7fcdb9d9d8f964d24a5ec3d423c67f49b7650ed8

I know it's a little bit late but the PR was merged recently

**Reply by cortinico** [reply]
For context: this is beneficial for users on Hermes on Windows. It would require a bump of the Gradle Plugin also. Safe to pick imho 👍 Not super urgent though.

**Reply by kelset** [reply]
when trying to cherry-pick this one, I ended up in a mess of merge conflicts; it seems that we'd need commits such as https://github.com/facebook/react-native/commit/6f7612252ba528f4fbabfe5c65b5bf775b776d28#diff-9ca1c97a718be1da688e30713ea3ec0cfd490a14eade4e61ceb98ef93ad04055 and https://github.com/facebook/react-native/pull/34298 and this second one in particular I don't think we want to introduce in a patch release of 0.69. So we'll not cherry-pick it. If you can open a PR to replicate the original change in the 0.69 branch we might consider it

**Comment by billnbell** [comment]
How about https://github.com/facebook/react-native/commit/70daa6e72822e3eeda4a1c4ebe9a7daa4fc2ac1f

**Reply by kelset** [reply]
I tried to cherry-pick this one but it doesn't merge cleanly and when I try to also pull in other commits to address this, things don't get any easier. So we'll not cherry-pick it. If you can open a PR to replicate the original change in the 0.69 branch we might consider it

**Comment by cortinico** [comment]
I think we should pick: 
https://github.com/facebook/react-native/commit/54a4fcbfdcc8111b3010b2c31ed3c1d48632ce4c

**Comment by kelset** [comment]
so that we won't forget https://github.com/facebook/react-native/pull/34439

**Comment by kelset** [comment]
another commit to add https://github.com/facebook/react-native/commit/78cd689f9a634b152ea09ed6cb4fa858ee26e653
